packages/ui-bindings/Agent/agent.tsx

```diff
@@ -51,12 +51,12 @@ query($token: String, $agentId: Int) {
         unplannedEconomicEvents {
           action
         }
-        processEconomicEvents {
-          action
-        }
-        processCommitments {
-          action
-        }
+        #processEconomicEvents {
+        #  action
+        #}
+        #processCommitments {
+        #  action
+        #}
```

Comment:
Commented out code is against coding style

Reply:
The naming had changed. I left the original in there for the time being so other developers who had not pulled in the lastest from api-extensions would know what might fix their code if they had errors.